README.md

```diff
@@ -378,7 +378,7 @@ method takes one argument:
 
 1. `b` must be an Apply of a function
 
-    1. If `b` does not represent a function, the behaviour of `ap` is
+    1. If `b` does not represent an Apply of a function, the behaviour of `ap` is
        unspecified.
```

Comment:
If I read _represent_ as _wrap_, the current wording seems reasonable.

This implies that there's exactly one function "inside", which is not necessarily the case: `[]` and `[Math.sin, Math.cos]` are straightforward counterexamples.